bitnami/redis-cluster/README.md

```diff
@@ -1,12 +1,12 @@
-<!--- app-name: Redis&trade; Cluster -->
+<!--- app-name: Redis&reg; Cluster -->
 
-# Redis(TM) Cluster packaged by Bitnami
+# Bitnami package for Redis(R) Cluster
```

Comment:
@carrodher is there any reason why we do not use &trade or &reg in the titles like this one?

Reply:
Those sections from the README (title and main description) are not hardcoded but filled with dynamic information coming from the Bitnami Admin panel.
Those descriptions and titles are reused across marketplaces and other places where the products are being distributed. `&trade` can work in markdown but not in other formats.

The rest of the README is not autogenerated, so in other sections those special characters can be used without any issue.

Reply:
Thanks for the information! :)